app/controllers/home_controller.rb

```diff
@@ -5,6 +5,11 @@ def index
     @data_stocks = load_categories_series(@categories)
   end
 
+  def update_stocks
```

Comment:
[![Documentation](https://www.pullreview.com/assets/documentation-small.png)](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-26) Add documentation to the following method[ `HomeController#update_stocks`](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-26)